src/ResourceTemplateValidator.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+/**
+ * Validates a resource template.
+ * Validation checks that does not contain repeated property templates.
+ * @param {<object>} resource template to validate
```

Comment:
Add the parameter name here. I believe the convention for jsdoc is to use `{Object}` (capitalized, no `<>`)